### DISCUSSION TITLE
Create and manage gene lists

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#2154] [Q&A] [answered]
Hello,

I have installed a local instance of seqr. I would like to know how works the creation and management of gene lists for users ? Can I create a gene list with my user, and another person could add genes in this list ? Which rights should I need to put for this user ?
Thank you for your help.

**Comment by hanars** [comment, marked as answer]
Once you create a gene list, only you will be able to edit it. If you make the list "public" any other user will be able to see the list. If you add a list to a project, regardless of whether the list is public, everyone with access to that project will be able to see the list. 

You should only have one user account for accessing seqr. Any user can create their own gene lists, so you do not have to do anything special to your user

**Reply by bnouyou** [reply]
Thank you for your answer. So if I create a gene list in "public", the other users can see it and use it, but they can't add other genes in it, right?

**Reply by hanars** [reply]
when you create a gene list, there is a checkbox you can select to indicate whether or not you want the gene list to be "public". If you make the list public, then yes everyone else can see the list but not edit it
